newcomers.html

```diff
@@ -50,7 +50,7 @@
         <li><a href="/index.html#programs">Programs</a></li>
         <li class="toggle"><a href="#main-menu">Contribute<span class="fas fa-chevron-down"></span></a>
             <ul>
-              <li><a href="/newcomers">Newcomers</a></li>
+              <li><a href="./newcomers.html">Newcomers</a></li>
```

Comment:
Are you using ecstatic (as explained in the readme) to view the website locally? If you are, this change shouldn't be necessary.